taxonomies/food/categories.txt

```diff
@@ -104618,7 +105490,7 @@ comment:en: has sugar grain size (0.35 mm) between powders and granulated
 
 # description:en:Granulated, familiar as table sugar, with a grain size about 0.5-0.6 mm across.
 # comment:en:"granulated sugar" is generally white sugar, but other sugars can be in granulated form too.
-< en:sugar
+< en:sugars
```

Comment:
Maybe an unintended change at this line - this parent category modification seems unrelated to frozen foods?

Reply:
I'll close if OK for you?